circle.yml

```diff
@@ -0,0 +1,24 @@
+machine:
+  xcode:
+    version: "8.0"
+  environment:
+    XCODE_WORKSPACE: NotUsed.xcworkspace
+    XCODE_SCHEME: NotUsed
+    GOPATH: $HOME/.go_project
+    HOMEBREW_NO_AUTO_UPDATE: 1
+
+dependencies:
+  pre:
+    - brew install https://github.com/Homebrew/homebrew-core/raw/master/Formula/go.rb
```

Comment:
oh, that's what that does? cool.

don't worry about it. it's at least documented in this pull request now. thanks!